llvm/test/CodeGen/WinEH/lit.local.cfg

```diff
@@ -1,11 +1,5 @@
-# FIXME: For now, override suffixes to exclude any .s tests, because some of the
-# buildbots have a stray misched-copy.s output file lying around that causes
-# failures. See misched-copy.s where we try and clean up that file.
-#
-# It should be possible to remove this override once all the bots have cycled
-# cleanly.
-config.suffixes = [".ll", ".test", ".txt"]
-
-# FIXME: Add Windows on ARM support to these tests.
-if not "X86" in config.root.targets:
+if not (
+    ("X86" in config.root.targets)
+    | ("AArch64" in config.root.targets)
+):
```

Comment:
This looks like a binary OR in what's supposedly a boolean context? Not really sure how Python behaves wrt that though.

Secondly, this would run the tests if either X86 or the AArch64 backend is enabled - but the tests use both, so this would cause failures if you have only one of them available.

Reply:
Right - that pattern is different there, as the JIT tests only execute for the native platform, so it's enough as long as the native arch is enabled as a target, and is one of the specific listed architectures.

Reply:
The form you pushed now should probably be fine; it's a bit more wordy, but it should work correctly. I'll leave it up to others to say which form you prefer. The former style, with the right condition in the lit.local.cfg file, is easier to read and write, but restricts the tests to only run on setups with both x86 and aarch64 targets enabled, while the new form requires more repeated code for the conditions, but makes the tests accessible to more build configurations.

Reply:
Agree it's more verbose but I didn't want to lose coverage for existing x86-only builders (no idea how many of those there are though).

This directory gets about 5/changes per year, so I don't think the extra complexity when writing tests is too big an issue.